docs/guide/pug-specific-options.md

```diff
@@ -0,0 +1,453 @@
+Sort attributes ascending.
+
+```pug
+Foo(a b c d)
+```
+
+#### `'desc'`
+
+Sort attributes descending.
+
+```pug
+Foo(d c b a)
+```
+
+---
+
+`pugSortAttributesBeginning`
+
+### Description
+
+Sort attributes by regex patterns to the beginning.
+
+### Options
+
+**Type:** `array`  
+**Default:** `[]`
+
+---
+
+`pugSortAttributesEnd`
+
+### Description
+
+Sort attributes by regex patterns to the end.
+
+### Options
+
+**Type:** `array`  
+**Default:** `[]`
+
+---
+
+[Example](https://github.com/prettier/plugin-pug/issues/22#issuecomment-699509995)
+
+---
+
+## Wrap Attributes
+
+`pugWrapAttributesPattern`
+
+### Description
+
+Define a regex pattern to match attributes against that should always trigger wrapping.
+
+### Options
+
+**Type:** `array`  
+**Default:** `[]`
+
+---
+
+`pugWrapAttributesThreshold`
+
+### Description
+
+Define the maximum amount of attributes that an element can appear with on one line before it gets wrapped.
+
+### Options
+
+**Type:** `choice`  
+**Default:** `-1`
+
+#### `-1`
+
+Only wrap attributes as needed.
+
+```pug
+input(type="text")
+input(type="text", value="my_value", name="my_name")
+```
+
+#### `0`
+
+Always wrap attributes.
+
+```pug
+input(
+  type="text"
+)
+input(
+  type="text",
+  value="my_value",
+  name="my_name"
+)
+```
+
+#### `1`
+
+Allow one unwrapped attribute, wrap two and more.
+
+```pug
+input(type="text")
+input(
+  type="text",
+  value="my_value",
+  name="my_name"
+)
+```
+
+#### `2 .. Infinity`
+
+Same as above, just with different thresholds.
+
+---
+
+## Framework specific
+
+`pugFramework`
+
+### Description
+
+Specify the used framework within the project.  
+If not set, or set to 'none', the plugin tries to find the correct framework by reading `process.env.npm_package_*`.
+
+### Options
+
+**Type:** `choice`  
+**Default:** `'none'`
+
+#### `'none'`
+
+Try to detect a framework by checking `process.env.npm_package_*`.  
+If no framework could be detected, the default strategy will be used.
+
+#### `'vue'`
+
+Set used framework to Vue.  
+This may hide some unrelated warnings that only apply to Angular.
+
+#### `'svelte'`
+
+Set used framework to Svelte.  
+_Currently behaves the same as Vue._  
+_If you found an issue with that, please open an [issue](https://github.com/prettier/plugin-pug/issues/new?template=bug-report.md)._
+
+#### `'angular'`
+
+Set used framework to Angular.  
+This uses a specific parser for text interpolation.
+
+---
+
+`pugSingleFileComponentIndentation`
+
+### Description
+
+Indent pug in template tags in single file components such as from vue or svelte.
+
+### Options
+
+**Type:** `boolean`  
+**Default:** `false`
+
+---
+
+## Empty Attributes
+
+`pugEmptyAttributes`
+
+### Description
+
+coming soon
+
+### Options
+
+**Type:** `choice`  
+**Default:** `'as-is'`
+
+Change behavior of boolean attributes.
+
+#### `'as-is'`
+
+Nothing is changed.
+
+```pug
+foo(a, b="", c)
+```
+
+#### `'none'`
+
+Every attribute with empty quotes will have them removed.
```

Comment:
```suggestion
Every boolean attribute with empty quotes will have them removed.
```

Or does this apply to all attributes?

```suggestion
Every attribute (regardless of type) with empty quotes will have them removed.
```

Reply:
This doesn't make sense theoretically cause HTML attributes are not typed at all